src/aws/Dockerfile

```diff
@@ -28,6 +31,56 @@ RUN npm install -g yarn
 # configure volume for lambda mediapackge
 VOLUME /mnt/transcoded_video
 
+WORKDIR /root
+
+RUN yum update -y && \
```

Comment:
no way to build static binary ? I'm not comfortable with the idea to push an image with development libraries.

Reply:
Once the build is done, maybe we can simply remove the packages needed for compilation ?

Reply:
I removed them, and tests still passes.
Let me know if it's ok for you, @lunika

Reply:
That's not ideal but I think it's enough